dropwizard-demo/test/com/test/TwitterImplementTest.java

```diff
@@ -0,0 +1,109 @@
+package com.test;
+
+import com.dep.Services.TwitterImplement;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import twitter4j.*;
+
+import javax.ws.rs.core.Response;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+
+@RunWith(MockitoJUnitRunner.class)
+public class TwitterImplementTest {
+    TwitterImplement twitterImplement;
+    TwitterFactory twitterFactory;
+    Twitter twitter;
+    Status status;
+
+    @Before
+    public void setUp() {
+        status = mock(Status.class);
+        twitterFactory = mock(TwitterFactory.class);
+        twitter = mock(Twitter.class);
+        when(twitterFactory.getInstance()).thenReturn(twitter);
+        twitterImplement = new TwitterImplement(twitterFactory);
+
+    }
+
+    @Test
+    public void testcase2_checkpost() throws TwitterException {
```

Comment:
I think you are handling the TwitterException it will not throw TwitterException (remove throws TwitterException)

Reply:
addressed reviews